fuzz/ParseFuzzer.cpp

```diff
@@ -0,0 +1,135 @@
+// Copyright 2020 Google LLC
+// Copyright 2025 Mikhail Khachaiants <mkhachaiants@gmail.com>
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "uriparser/Uri.h"
+#include "uriparser/UriIp4.h"
+#include "FuzzingUtils.h"
+#include <cstddef>
+#include <cstring>
+#include <vector>
+
+
+
+class UriHolder {
+public:
+	UriHolder() {
+		memset((void *)&uri_, 0, sizeof(uri_));
+	}
+
+	~UriHolder() {
+		URI_FUNC(FreeUriMembers)(&uri_);
+	}
+
+	URI_TYPE(Uri) * get() {
+		return &uri_;
+	}
+
+private:
+	URI_TYPE(Uri) uri_;
+};
+
+
+
+void Escapes(const UriString & uri) {
+	const URI_CHAR * first = uri.c_str();
+	// Up to 6 bytes per character with normalizeBreaks enabled (\\n -> %0D%0A)
```

Comment:
```suggestion
	// Up to 6 bytes per character with normalizeBreaks enabled (\n -> %0D%0A)
```
The double backslash seems mistaken (or I didn't get the idea)

Reply:
Fixed